components/mixins/experience-event/events/visitwebpage.schema.json

```diff
@@ -0,0 +1,134 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/visitwebpage",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Visit WebPage",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "visitwebpage",  
+  "meta:intendedToExtend": [
+  "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Use for information regarding web details events such as interaction, page details, and referrer.",
+  "definitions": {
+    "visitwebpage": {
+      "properties": {
+        "xdm:web": {
+          "title": "Web",
+          "type": "object",
+          "properties": {
+            "xdm:webPageDetails": {
+              "title": "Web Page Details",
+              "type": "object",
+              "properties": {
+                "xdm:name": {
+                  "title": "Name",
```

Comment:
Assume this is "Webpage ID" from Marketo side.
Let's capture this mapping in our wiki to help facilitate the discussion - whether to keep id and/or name for an object reference.